doc/examples/register_binary_fuzzy.py

```diff
@@ -0,0 +1,166 @@
+"""
+=========================================================
+Diffeomorphic Registration with binary and fuzzy images
+=========================================================
+This example for registering binary and fuzzy images together.
+This may be seen as aligning sensed (fuzzy) image to the
```

Comment:
Typo: "to the and" 

How about rewriting these two sentences as. 

"This example demonstrates registration of a binary and a fuzzy image. This could be seen as aligning a fuzzy (sensed) image to a binary (e.g., template) image."